ui/raidboss/data/06-ew/raid/p6s.ts

```diff
@@ -85,6 +95,259 @@ const triggerSet: TriggerSet<Data> = {
         },
       },
     },
+    {
+      id: 'P6S Polyominoid Tether Collect',
+      type: 'Tether',
+      netRegex: NetRegexes.tether({ id: '00CF' }),
+      run: (data, matches) => {
+        data.tileTethers.push(matches);
+      },
+    },
+    {
+      id: 'P6S Polyominoid MapEffect Collect',
+      type: 'MapEffect',
+      netRegex: NetRegexes.mapEffect({ flags: ['00020001', '00400020'] }),
+      run: (data, matches) => {
+        // location '00' is center/mapwide and won't be used
+        // for determining mechanic resolution
```

Comment:
More specifically, location `00` triggers the black lines outlining the tiles in the arena.

Reply:
Thanks - I never paid attention to that,  Will update the comment.